### ISSUE TITLE
Abnormal overfitting on QaTa-COV19 Dataset

### ISSUE THREAD

**Issue by AlexTS1980** [matterport/Mask_RCNN#2665]
Dataset contains 2951 CXR images with masks corresponding to lesion areas (1 positive class). I use 1000 as training split, the rest is testing, and I keep getting a very low mAP, around 0.2925, with 0% precision for IoU=0.95 

Previously I used CNCB-NCOV dataset for lesion segmentation (3 positive classes: clean lungs, GGO, C) in CTs, 650 for training, the rest testing, and I kept getting very good results  (0.4245 mean average precision, main MS COCO criterion). 

I decided to evaluate the model on the training data, and I found out that there's abnormal overfitting, something like 0.99 mAP. 

Any suggestions on what could be done about this? I reduced lrate to 1e-6, resampled that training/test splits several times, ran the model for a different number of epochs? changed the anchors, and the number of output/FPN  layers, nothing's working.
